site/pages/docs/ref/multimedia/multimedia-en.hbs

```diff
@@ -28,6 +28,16 @@
 	</ul>
 </section>
 
+<section>
+<h2>Evaluation and report</h2>
+<ul>
+	<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-audio-en.html">English Audio Only Report</a></li>
+	<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-audio-fr.html">French Audio Only Report</a></li>
+	<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-video-en.html">English Video Report</a></li>
+	<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-video-fr.html">French Video Report</a></li>
```

Comment:
Le caractère `&` dois être échapé. `&amp;`

Reply:
Addressed in commit "Updated assessment format"